libs/blocks/merch/merch.js

```diff
@@ -349,7 +377,13 @@ export async function initService(force = false) {
   const { env, commerce = {}, locale } = getConfig();
   commerce.priceLiteralsPromise = fetchLiterals(PRICE_LITERALS_URL);
   initService.promise = initService.promise ?? polyfills().then(async () => {
-    const commerceLib = await import('../../deps/commerce.js');
+    const { hostname, searchParams } = new URL(window.location.href);
+    const maslibs = searchParams.get('maslibs');
+    let commerceLibPath = '../../deps/commerce.js';
+    if (maslibs) {
+      commerceLibPath = `${getMasBase(hostname, maslibs)}/lib/commerce.js`;
+    }
```

Comment:
```suggestion
    const commerceLibPath = `${!maslibs ? '../../deps' : `${getMasBase(hostname, maslibs)}/lib`}/commerce.js`;
```

Reply:
I thought about it but I find it harder to read.  Most of the time, `maslibs` will be `false` and the whole expression will be skipped.